src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Exports/ExportProviderBuilder.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
This class is almost entirely taken from Roslyn with a few modifications to make it work for Razor